.github/workflows/nginx-bot.yml

```diff
@@ -0,0 +1,36 @@
+name: NGINX Bot
+
+on:
+  issues:
+    types:
+      - opened
+      - reopened
+      - edited
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - edited
+
+permissions:
+  contents: read
+  issues: write
+  pull-requests: write
+
+jobs:
+  comment:
+    name: Run the bot
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        id: checkout
+        uses: actions/checkout@v4
+
+      - name: Bot Action
+        uses: lucacome/nginx-bot@main
```

Comment:
This is just to test it out, it will be moved to nginx org and versioned in the future.